cockatrice/src/gameselector.cpp

```diff
@@ -212,3 +216,17 @@ void GameSelector::processGameInfo(const ServerInfo_Game &info)
 {
     gameListModel->updateGameList(info);
 }
+
+void GameSelector::updateButtonChoices(const QModelIndex &index)
+{
+    if (!index.isValid())
+        return;
+
+    const ServerInfo_Game &game = gameListModel->getGame(index.data(Qt::UserRole).toInt());
+    bool overrideRestrictions = !tabSupervisor->getAdminLocked();
```

Comment:
Ok, so you need to click a button to enable a superuser permission.
This line also assigns an inverse result. It returns the `getAdminLocked()` and then `!` it. This doesnt feel right. The `!` should be placed in the check.